core/src/main/java/org/hibernate/ogm/index/OgmIndex.java

```diff
@@ -0,0 +1,25 @@
+/*
+ * Hibernate OGM, Domain model persistence for NoSQL datastores
+ *
+ * License: GNU Lesser General Public License (LGPL), version 2.1 or later
+ * See the lgpl.txt file in the root directory or <http://www.gnu.org/licenses/lgpl-2.1.html>.
+ */
+package org.hibernate.ogm.index;
+
+import java.lang.annotation.Retention;
+import java.lang.annotation.Target;
+
+import static java.lang.annotation.ElementType.ANNOTATION_TYPE;
+import static java.lang.annotation.RetentionPolicy.RUNTIME;
+
+/**
+ * MetaAnnotation for various Index related annotation
+ *
+ * @author Francois Le Droff
+ */
+@Target({ ANNOTATION_TYPE })
+@Retention(RUNTIME)
+public @interface OgmIndex {
```

Comment:
dead code from my first iteration, see the first comments and the first commit.
I will clean that up.